app-office/libreoffice/libreoffice-5.1.4.2.ebuild

```diff
@@ -265,7 +265,7 @@ pkg_pretend() {
 	if [[ ${MERGE_TYPE} != binary ]]; then
 		check-reqs_pkg_pretend
 
-		if [[ $(tc-getCC) == clang ]] ; then
+		if [[ $(tc-getCC) == *clang* ]] ; then
```

Comment:
I'd much rather check for the exception on top (clang being used) and then have it default to GCC for whatever may go wrong there.